### ISSUE TITLE
configure for Dark Panel and White Windows theme

### ISSUE THREAD

**Issue by twnaing** [keeferrourke/la-capitaine-icon-theme#108]
Could you please make it possible to configure for Dark Panel & White Windows (e.g. Greybird theme) theme?

**Comment by twnaing** [comment]
Currently I do the following to achieve what I want.

1. answer no to dark theme
2. go to `~/.icons/la-capitaine-icon-theme/panel` and delete symbolic linked `16` and `24` folders
3. manually run 

```bash
ln -sf ~/.icons/la-capitaine-icon-theme/panel/16-dark ~/.icons/la-capitaine-icon-theme/panel/16
ln -sf ~/.icons/la-capitaine-icon-theme/panel/24-dark ~/.icons/la-capitaine-icon-theme/panel/24
```

**Comment by keeferrourke** [comment]
Sure thing, I can make the panel symlinking a separate prompt. Update should be in git sometime tonight, if it's not up by tomorrow ping me.

**Comment by keeferrourke** [comment]
**Clearly** this issue fell by the wayside. My apologies. Fixed as of commit 2e096b3.
